dw-icon-button.js

```diff
@@ -140,19 +137,9 @@ export class DwIconButton extends buttonFocus(LitElement) {
     return styleMap({ width: this.buttonSize + 'px', height: this.buttonSize + 'px', padding: padding + 'px'});
   }
 
-  _onClick() {
-    /**
-    * call blur method to fix ripple effect after icon click.
-    */
-    setTimeout(() => {
-      this.shadowRoot.querySelector('button').blur();
-    }, 350);
-  }
-
   constructor() {
     super();
     this.disabled = false;
-    this.active = false;
```

Comment:
I don't understand this change either.

Reply:
This was pre-planned. As `active` property is interfering with the `active` property implementation of our custom `LitElement`. So, we are removing this feature.